### ISSUE TITLE
Sensors: Create a data template for real-time sensor data 

### ISSUE THREAD

**Issue by aislinnpearson** [Rothamsted-Ecoinformatics/farm_rothamsted#255]
Following a discussion with Had, we raised the idea that there should be a standard processing step which takes sensor data and coverts it into a standard format that can be easily read into a database by @GaryFrewin. This relates to a previous discussion with @richardostler, @GaryFrewin and myself in terms of a data standard for real-time data. 

The first step is getting the sensors into FarmOS (#253) and giving them a geometry (#252). Once this is done, I think there needs to be an internal discussion abut what this data standard should look like and possibly identifying a few examples on the North Wyke Farm Platform for developing an initial standard. I doubt that the data transformation will happen in FarmOS itself, but I know that standardisation is something the OpenTEAM community is working on so I wanted to include an issue here for us to refer to. Especially as I imagine FarmOS will serve as a useful catalogue of the sensors already in place, and where the data is stored.

Marked as nice to have because, although a useful outcome of the digital transformation, it's not currently a priority for FarmOS development.
